### PR TITLE
feature/EEMD-182_text_field_date

### PR DESCRIPTION
**Pull request by juantobonq**
UI Test:
image

https://user-images.githubusercontent.com/101196521/191140383-9bc66558-cb61-483c-a97f-03b87baf5c04.mp4

### CODE REVIEW

File: demo/src/main/java/co/condorlabs/demo/screens/samples/TextFieldScreen.kt [hunk 1]
```diff
@@ -62,10 +70,37 @@ fun TextFieldScreenSample() {
...
+    min.set(Calendar.YEAR, 2020)
+    min.set(Calendar.MONTH, 1)
+    min.set(Calendar.DAY_OF_MONTH, 1)
+
+    val max = Calendar.getInstance()
+    max.set(Calendar.YEAR, 2022)
+    max.set(Calendar.MONTH, 11)
+    max.set(Calendar.DAY_OF_MONTH, 31)
+
     Column(modifier = Modifier.padding(10.dp)) {
```
Comment (JhoanGamarra) [inline demo/src/main/java/co/condorlabs/demo/screens/samples/TextFieldScreen.kt:104]: use dimens please

**Comment by JhoanGamarra** [comment]
The keyboard is overlapping the Date field, try to fix that.

File: app/src/main/java/co/condorlabs/emerald/components/textfield/EmeraldDateTextField.kt [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    val mCalendar = Calendar.getInstance()
+    mCalendar.time = date
+
+    // Fetching current year, month and day
+    mYear = mCalendar.get(Calendar.YEAR)
+    mMonth = mCalendar.get(Calendar.MONTH)
+    mDay = mCalendar.get(Calendar.DAY_OF_MONTH)
+
+
+    // Declaring DatePickerDialog and setting
```
[the remark this answers is not in the document]
Comment (juantobonq) [reply app/src/main/java/co/condorlabs/emerald/components/textfield/EmeraldDateTextField.kt:71]: Done

File: demo/src/main/java/co/condorlabs/demo/screens/samples/TextFieldScreen.kt (hunk 1 above)
Comment (juantobonq) [reply demo/src/main/java/co/condorlabs/demo/screens/samples/TextFieldScreen.kt:104]: Done

**Review by leidyzulu — CHANGES_REQUESTED** [review]

File: app/src/main/java/co/condorlabs/emerald/components/textfield/EmeraldDateTextField.kt [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.KeyboardType
+import co.condorlabs.emerald.R
+import co.condorlabs.emerald.components.utils.Empty
+import java.util.Calendar
+import java.util.Date
+
+@Composable
+fun EmeraldDateTextField(
```
[the remark this answers is not in the document]
Comment (juantobonq) [reply app/src/main/java/co/condorlabs/emerald/components/textfield/EmeraldDateTextField.kt:29]: Done

**Review by leidyzulu — APPROVED** [review]

**Review by JhoanGamarra — APPROVED** [review]

**Pull request merged**